part-two/Attention.md

```diff
@@ -2,9 +2,9 @@
 
 There is a limited amount of human attention in the world. We have 24 hours in the day and we need to spend some of that time eating and sleeping. For many people in the world much of their waking time is occupied with work. So that leaves relatively little time for attention that we can freely allocate. This hard limit also exists in the aggregate, since&mdash;as I have argued earlier&mdash;we are headed for peak population.
 
-At the same time that our attention is limited, we are using the internet to dramatically increase the amount of available content. The increase in content is well documented to be exponential, which means that most of the content that has ever been produced by humanity has been produced in the last few years. For example, Youtube alone is adding 100 hours of new video content every minute {{ "TNW" | cite}}.
+At the same time that our attention is limited, we are using the Internet to dramatically increase the amount of available content. The increase in content is well documented to be exponential, which means that most of the content that has ever been produced by humanity has been produced in the last few years {{"FORBES2" | cite}}. For example, YouTube alone is adding 100 hours of new video content every minute {{ "TNW" | cite}}.
 
-As a result, it is easy today to be completely overwhelmed by content. Our limited attention can readily be absorbed by ever refreshing content. Humans are maladapted to the information environment we now live in. It is analogous to having an obsession for sweets in an environment that is now sugar rich. Checking email, Twitter, Instagram, watching yet another Youtube clip or Snapchat story, or episode of one&apos;s favorite show on a streaming service&mdash;these all provide quick &ldquo;information hits&rdquo; that trigger parts of our brain attuned to novelty.  
+As a result, it is easy today to be completely overwhelmed by content. Our limited attention can readily be absorbed by ever refreshing content. Humans are maladapted to the information environment we now live in. It is analogous to having an obsession for sweets in an environment that is now sugar rich. Checking email, Twitter, Instagram, watching yet another YouTube clip or Snapchat story, or episode of one&apos;s favorite show on a streaming service&mdash;these all provide quick &ldquo;information hits&rdquo; that trigger parts of our brain attuned to novelty. As of 2017, the average person spends roughly two hours on social media every day, and this does not even include the time we spend watching TV {{"SMT" |cite}}. 
```

Comment:
Please delete the additional ", and this does not even include the time we spend watching TV" ∂